src/sentry/integrations/msteams/utils.py

```diff
@@ -463,6 +469,98 @@ def build_group_card(group, event, rules):
     return {"type": "AdaptiveCard", "body": body}
 
 
+def build_incident_attachment(incident, metric_value=None):
+    title, text, logo_url, status, ts = incident_attatchment_info(incident, metric_value)
+
+    colors = {"Resolved": "good", "Warning": "warning", "Critical": "attention"}
```

Comment:
@NisanthanNanthakumar The Slack color lookup has 5 different color levels. I realize in the UI there are only 3 but we should verify that there aren't plans to add the other levels any time soon.

Reply:
We don't, but help me understand what would happen if we did?

Reply:
@adhiraj we'd have to update integration specific code for the colors (and probably other things) for it to work